Build/Unicorn.nuget/Unicorn.nuspec

```diff
@@ -12,15 +12,35 @@
 		<description>Unicorn is a utility for Sitecore that solves the issue of moving templates, renderings, and other database items between Sitecore instances. This package contains both the Unicorn core library and the configuration for it, which is appropriate for web projects. Install Unicorn.Core if you only want the library.</description>
 		<copyright>Copyright 2018</copyright>
 		<tags>sitecore serialization</tags>
-	<dependencies>
-		<dependency id="Unicorn.Core" version="$version$" />
-		<dependency id="Rainbow" version="$rainbowversion$" />
-	</dependencies>
+		<dependencies>
+			<dependency id="Unicorn.Core" version="$version$" />
+			<dependency id="Rainbow" version="$rainbowversion$" />
+		</dependencies>
+		<!--
+			PackageReference support
+			Describes how assets will be restored whent the pacakge is consumed
+			We want all assets to be used as content files
+			The include path is rooted to the contentFiles path
+		-->
+		<contentFiles>
+			<files include="any/**/*" buildAction="Content" copyToOutput="true" />
+		</contentFiles>
 	</metadata>
 	<files>
+		<!--
+			PackageReference support
+			contentFiles\{language}\{tfm}
+			lanuguage - cs, vb etc. We use any as the assets are not language specific
+			tfm - net462,netstandard2.0 etc - We use any as the assets are not runtime specific
+		-->
+		<file src="..\..\src\Unicorn\Standard Config Files\*.config" target="contentFiles\any\any\App_Config\Include\Unicorn" />
```

Comment:
Same include all the files, theoretically shouldn't need each extension enumerated. I think, anyway.